### DISCUSSION TITLE
Could not find any matches for ionic-team:capacitor-android:1+

### DISCUSSION THREAD

**Discussion by moez82** [ionic-team/capacitor#5130] [Q&A] [answered]
Hi everybody,

I'm stuck since few days on this error when building my project in Android Studio

Execution failed for task ':capacitor-image-crop:generateDebugRFile'.
  Could not resolve all files for configuration ':capacitor-image-crop:debugCompileClasspath'.
    Could not find any matches for ionic-team:capacitor-android:1+ as no versions of ionic-team:capacitor-android are available.
     Required by:
         project :capacitor-image-crop

Any help will be welcome :)
Thank you.

**Comment by jcesarmobile** [comment]
That plugin is for capacitor 1 and has not been updated for capacitor 3, so you won’t be able to use it.

**Comment by moez82** [comment, marked as answer]
I found the solution to my problem, I post it here in case someone else will face it.
To resolve the error we must change  :  **implementation 'ionic-team:capacitor-android:1+'**
to this : **implementation project(':capacitor-android')**
